### PR TITLE
🏇 Feed logic tests

### LINKED ISSUE #86: 🏇  Feed: Add Main Feed Logic Tests
The main feed function needs to be tested.

Add a tests to check the following:
- The main feed function returns a QuerySet
- Each element in the QuerySet is a Post
- Iterate through the posts and compare their attributes to the data migration posts

### PR DESCRIPTION
**Pull request by DeanBiton**
Added tests to check main feed function.

Tests aim to check:
- The main feed function returns a QuerySet
- Each element in the QuerySet is a Post
- Each post in the QuerySet compared to the data migration posts

Please notice that the first 2 commits are from an earlier PR.
Dependes on #64
Close #86

### CODE REVIEW

**Review by paOmer — APPROVED** [review]

File: feed/tests/test_comment_model.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+@pytest.fixture
+def comments(post0, users):
+    comment0 = Comment(author=users[1], post_parent=post0, content=COMMENT1_CONTENT)
+    comment1 = Comment(author=users[0], post_parent=post0, comment_parent=comment0, content=COMMENT2_CONTENT)
+    return [comment0, comment1]
+
+
+@pytest.mark.django_db
```
Comment (Yarboa) [inline feed/tests/test_comment_model.py:34]: Please consult with @paOmer about how to treat the tests and fixtures
see this as reference tests #64

**Review by Yarboa — CHANGES_REQUESTED** [review]
see generic comment above

**Review by Yarboa — APPROVED** [review]
LGTM

**Review by ShirleyFichman — APPROVED** [review]
Great small PR, LGTM

**Pull request merged**